pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -206,6 +297,17 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         action: () => subvolume_create(volume, subvol, (mounted && !opt_ro) ? mount_point : mount_point_in_parent),
     });
 
+    let delete_excuse = create_excuse;
+    if (subvol.id === 5)
+        delete_excuse = _("Deleting the root subvolume is not allowed");
+
+    actions.push({
+        danger: true,
+        title: _("Delete subvolume"),
```

Comment:
I think it's better to just call this "Delete".  With "Delete subvolume", you have "Create subvolume" and "Delete subvolume" next to each other, but they refer to different things. The first will create a new subvolume of this subvolume, the latter will delete this very subvolume (and not one of its children).

Reply:
Also, we don't say "Mount subvolume", for example.